trieste/models/model_interfaces.py

```diff
@@ -169,8 +169,8 @@ class GPflowPredictor(ModelInterface, ABC):
     def model(self) -> GPModel:
         """ The underlying GPflow model. """
 
-    def predict(self, query_points: QueryPoints) -> Tuple[ObserverEvaluations, TensorType]:
-        return self.model.predict_f(query_points)
+    def predict(self, query_points: QueryPoints, full_cov: bool = False) -> Tuple[ObserverEvaluations, TensorType]:
+        return self.model.predict_f(query_points, full_cov=full_cov, full_output_cov=False)
```

Comment:
```suggestion
        return self.model.predict_f(query_points, full_cov=full_cov)
```
`full_output_cov` defaults to `False`